CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+# v0.10.0 (*08.08.2024*)
+
+## Major Changes since v0.9.0
+* Automatic network generation from high-level specifications.
+* Units at the user interface including scaling and conversion.
+* Morphologies are now loaded through a unified interface, bundling morphology, meta data, and segment tree.
```

Comment:
this sentence reads odd to me

Reply:
also, maybe use 'metadata' to be consistent?

Reply:
Yes